build/charts/flow-aggregator/conf/flow-aggregator.conf

```diff
@@ -72,9 +72,23 @@ clickHouse:
   # Database is the name of database where Antrea "flows" table is created.
   database: "default"
 
-  # DatabaseURL is the url to the database. TCP protocol is required.
+  # DatabaseURL is the url to the database. TCP/HTTP/HTTPS protocols are supported.
   databaseURL: {{ .Values.clickHouse.databaseURL | quote }}
 
+  # tls contains tls related configuration options, which is used to connect to clickhouse service.
+  tls:
+    # Enable is the switch to enable TLS when connecting to ClickHouse server.
+    enable: {{ .Values.clickHouse.tls.enable }}
```

Comment:
I think this can be removed. So that we can get rid of the HTTP+tlsEnabled -> error, HTTPS+tlsEnabled -> error statements in clickhouseclient.go.

Reply:
But how about tcp? We can still enable/disable tls for tcp?

Reply:
Updated. Thanks.